pkg/prober/packet.go

```diff
@@ -58,7 +52,7 @@ func (p *Prober) craftPacket(pr *probe) ([]byte, error) {
 
 	// Create final UDP packet that will return
 	ip := &layers.IPv4{
-		SrcIP:    innerSrc,
+		SrcIP:    p.hops[len(p.hops)-1].srcRange[pr.Seq%uint64(len(p.hops[len(p.hops)-1].srcRange))],
```

Comment:
Thats a magic line. Can we somehow define this as a function on Prober? To give the reader some hint what this is all about.

Reply:
Done.